not-to-release/sources/newsgroup/groups.google.com_alt.animals.cat_01ff709c4bf2c60c_ENG_20040418_040100.xml.conllu

```diff
@@ -1019,8 +1019,8 @@
 15	as	as	SCONJ	IN	_	18	mark	18:mark	_
 16	they	they	PRON	PRP	Case=Nom|Number=Plur|Person=3|PronType=Prs	18	nsubj	18:nsubj	_
 17	can	can	AUX	MD	VerbForm=Fin	18	aux	18:aux	_
-18	surv	surv	VERB	GW	_	14	advcl	14:advcl:as	_
-19	ive	ive	X	VB	VerbForm=Inf	18	goeswith	18:goeswith	SpaceAfter=No
+18	surv	survive	VERB	GW	Typo=Yes|VerbForm=Inf	14	advcl	14:advcl:as	_
+19	ive	_	X	VB	_	18	goeswith	18:goeswith	SpaceAfter=No
```

Comment:
Should UD take a position on XPOS? It seems the tagger convention was to put the linguistic XPOS on the last token in the `goeswith` group and `GW` for the rest, the opposite of all the other info. Would be nicer to search if the meaningful XPOS was also on the first token but maybe that would interfere with PTB tagger compatibility.

Reply:
I think we discussed XPOS. @amir-zeldes said that UD should not take a position on XPOS, which I agree with, as XPOS is mostly a non-UD thing (except that UD provides a space where XPOS can be shown).